x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/LifecycleExecutionState.java

```diff
@@ -259,6 +269,7 @@ public int hashCode() {
         private Long phaseTime;
         private Long actionTime;
         private Long stepTime;
+        private Long indexOriginationDate;
```

Comment:
As the `index.creation_date` and `index.lifecycle.origination_date` settings have default values, would it be worth working with primitives here?